### PR TITLE
try to deploy fate serving admin

### PR DESCRIPTION
**Pull request by asdfsx**
add fate serving admin deploy script

### CODE REVIEW

File: helm-charts/FATE-Serving/templates/serving-admin-module.yaml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  labels:
+    fateMoudle: serving-admin
+    name: {{ .Values.partyName | quote  }}
+    partyId: {{ .Values.partyId | quote  }}
+    owner: kubefate
+    cluster: fate-serving
+data:
+  application.properties: |
+    #
+    # Copyright 2019 The FATE Authors. All Rights Reserved.
```
Comment (jiahaoc1993) [inline helm-charts/FATE-Serving/templates/serving-admin-module.yaml:26]: It is fine to remove the header.

Comment (asdfsx) [reply helm-charts/FATE-Serving/templates/serving-admin-module.yaml]: header removed

**Review by jiahaoc1993 — COMMENTED** [review]
LGTM

**Comment by jiahaoc1993** [comment]
@owlet42 hi chenlong, could you review this PR?

**Review by owlet42 — APPROVED** [review]

**Comment by owlet42** [comment]
@asdfsx All PR needs to pass the DCO verification.
d11b073‘s DCO check was not successful.

**Comment by asdfsx** [comment]
@owlet42 DCO check updated

**Pull request merged**